app/AndroidManifest.xml

```diff
@@ -374,7 +374,9 @@
         <activity android:name="org.commcare.activities.MessageActivity">
         </activity>
 
-        <receiver android:name="org.commcare.views.notifications.NotificationClearReceiver">
+        <receiver
+            android:exported="false"
+            android:name="org.commcare.views.notifications.NotificationClearReceiver">
```

Comment:
we should either add a qa note to test clearning of notifications or test it out ourselves and mention it in safety story of the PR

Reply:
I tested this and it worked fine, I will add a note to the PR.